### DISCUSSION TITLE
fundMe (VM Execution error.)

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#618] [Q&A] [answered]
Hello,
I am in lesson 3 of PatrickCollins' course on freeCodeCamp where we make getPrice function to return the latest price of Ethereum. 
I deployed my contract, i have the getPrice button in red. 

But when I click on it it does not return "uint256" but an error in the console which says

```
call to FundMe.getPrice errored: VM execution error.
Reverted0x
```
----
Do you know why ? 
--------
```Solidity
function fund() public payable {
    addressToAmountFunded[msg.sender] += msg.value;
    // what the ETH -> USD conversion rate
}

function getVersion() public view returns (uint256) {
    AggregatorV3Interface priceFeed = AggregatorV3Interface(0x8A753747A1Fa494EC906cE90E9f37563A8AF630e);
    return priceFeed.version();
}

function getPrice() public view returns(uint256) {
  AggregatorV3Interface priceFeed = AggregatorV3Interface(0x8A753747A1Fa494EC906cE90E9f37563A8AF630e);
  (
    uint80 roundId,
    int256 answer,
    uint256 startedAt,
    uint256 updatedAt,
    uint80 answeredInRound) 
    = priceFeed.latestRoundData();
    return uint256(answer);
}

```

**Comment by cromewar** [comment, marked as answer]
Hello @NinjaTurtles-cloud 
This address: 0x8A753747A1Fa494EC906cE90E9f37563A8AF630e is for RInkeby test network, so be sure you are using Rinkeby on your metamask.
Also could you please share the entire script so we could check it?

**Reply by wilonweb** [reply]
Yeah it was beacause i was on Kovan from my metamask
Thanks you !!!

**Reply by cromewar** [reply]
I'm glad this info helped, closing this thread now.
